### PR TITLE
Feature 32 - Add tooltips to the forms

### LINKED ISSUE #32: Feature - Add tooltips to the forms
Maybe the user doesn't know the difference between the english levels or the seniority levels so we can give them an idea about it and a link to have more info. Also, the compare button should only be enabled when all the data is filled
## Checklist ✅

- [x] The issue is present in the latest release.
- [x] I have searched the [issues](https://github.com/Platzi-Master-C8/gethired-base/issues) of this repository and believe that this is not a duplicate.

## Summary 💡
In the for there should be a couple of icons that when you click the, it shows a tooltip with relevant information and a link to get more info. The user should no make a request to the API until all the info have been filled

## Requirements 🌈
It should help the user to understand the difference between levels

## Motivation 🔦

Improve the user experience

### PR DESCRIPTION
**Pull request by johannpereze**
## Checklist

- [x] I have followed the [contributor guideline](https://github.com/Platzi-Master-C8/gethired-base/blob/main/CONTRIBUTING.md)
- [ ] Tests are included
- [ ] Documentation is changed or added
- [x] Related issue has been created
- [x] Commits messages follows [commit guideline](https://github.com/Platzi-Master-C8/gethired-base/blob/main/CONTRIBUTING.md/#Commits)

### Type of PR

- [x] Feature
- [ ] Bugfix
- [ ] Hotfix
- [ ] Chore

### Merge strategy

- [x] Squash (default)
- [ ] Merge commit (branch merges, usually used for merges to master)
- [ ] Rebase (preserve individual commits without merge commit)

### Purpose
Maybe the user doesn't know the difference between the english levels or the seniority levels so we can give them an idea about it and a link to have more info. Also, the compare button should only be enabled when all the data is filled
> Enlist the purposes or write an abastract about it, please be detailed, remember that not all the members of the team have the context of yout tasks
### Relevant tickets

> Paste here the link to the task ticket
resolves [#32]

### Related PR's

> Paste here the links to related PR's
### Screenshots

> Add screenshots that could help others do understand even more the objective of your pr

### CODE REVIEW

File: src/components/Commons/InfoTooltip/InfoTooltip.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    const handleTooltipClose = () => {
+        setOpen(false);
+    };
+
+    const handleTooltipOpen = () => {
+        setOpen(true);
+    };
+
+    return (
+        <div>
```
Comment (kevfarid) [inline src/components/Commons/InfoTooltip/InfoTooltip.js]: Delete this div, replace it with a fragment

File: src/components/Commons/InfoTooltip/InfoTooltip.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+            <ClickAwayListener onClickAway={handleTooltipClose}>
+                <div>
```
Comment (kevfarid) [inline src/components/Commons/InfoTooltip/InfoTooltip.js]: Delete this div, this is not necessary.

File: src/constants/index.js [hunk 3]
```diff
@@ -29,3 +29,27 @@ export const values2 = [
 ];
 
 export const currencyName = 'USD';
+
```
Comment (kevfarid) [inline src/constants/index.js:32]: This must function as a service

File: src/components/FormCard/FormCard.jsx [hunk 4]
```diff
@@ -11,6 +11,8 @@ import { selectTechnologies, selectJobs, selectSeniority } from 'App/ListData/se
 import { fetchListData } from 'App/ListData/slice';
 
 import { getListByName } from 'Services/salaries';
+import { englishInfo, seniorityInfo } from 'Constants';
```
Comment (kevfarid) [inline src/components/FormCard/FormCard.jsx]: listSenority is the same that seniorityInfo.
english_level is the same that englishInfo.

**Review by kevfarid — CHANGES_REQUESTED** [review]

File: src/constants/index.js [hunk 5]
```diff
@@ -29,3 +29,27 @@ export const values2 = [
 ];
 
 export const currencyName = 'USD';
+
+export const englishInfo = {
```
Comment (kevfarid) [inline src/constants/index.js]: This must function as a service

**Review by kevfarid — CHANGES_REQUESTED** [review]

File: src/constants/index.js (hunk 5 above)
Comment (johannpereze) [reply src/constants/index.js]: It's temporary until the backend team develops the endpoint

Comment (johannpereze) [reply src/constants/index.js]: Should I add a comment that explains it?

File: src/components/Commons/InfoTooltip/InfoTooltip.js (hunk 1 above)
Comment (johannpereze) [reply src/components/Commons/InfoTooltip/InfoTooltip.js]: Done. It contained the click away listener as the parent tag for all. So I just deleted the div.

File: src/components/Commons/InfoTooltip/InfoTooltip.js (hunk 2 above)
Comment (johannpereze) [reply src/components/Commons/InfoTooltip/InfoTooltip.js]: Done

File: src/components/FormCard/FormCard.jsx (hunk 4 above)
Comment (johannpereze) [reply src/components/FormCard/FormCard.jsx]: export const listSenority = ['Junior', 'Mid-Junior', 'Mid', 'Mid-Senior', 'Senior'];
export const ListEnglish = ['A1', 'A2', 'B1', 'B2', 'C1', 'C2'];

They're not the same shape. listSenority and ListEnglish are not being used. I will delete them from Constants

File: src/constants/index.js (hunk 5 above)
Comment (johannpereze) [reply src/constants/index.js]: I added the comment

File: src/components/FormCard/FormCard.jsx [hunk 6]
```diff
@@ -11,6 +11,8 @@ import { selectTechnologies, selectJobs, selectSeniority } from 'App/ListData/se
 import { fetchListData } from 'App/ListData/slice';
 
 import { getListByName } from 'Services/salaries';
+import { englishInfo, seniorityInfo } from 'Constants';
+import { InfoTooltip } from 'Components/Commons/InfoTooltip/InfoTooltip';
```
Comment (yadurani) [inline src/components/FormCard/FormCard.jsx]: Move this components line 8

**Review by yadurani — CHANGES_REQUESTED** [review]

**Review by kevfarid — APPROVED** [review]
Nice, good job

File: src/services/salaries.js [hunk 7]
```diff
@@ -4,7 +4,29 @@ const API_URL = process.env.SALARIES_API_URL;
...
+                { level: 'A1', description: 'Can understand and use familiar everyday expressions and very basic phrases.' },
+                { level: 'A2', description: 'Can understand sentences and frequently used expressions.' },
+                { level: 'B1', description: 'Can deal with most situations likely to arise while travelling.' },
+                { level: 'B2', description: 'Can interact with a degree of fluency with native speakers.' },
+                { level: 'C1', description: 'Can use language effectively for social, academic and professional purposes.' },
+                { level: 'C2', description: 'Can understand with ease virtually everything heard or read.' }
+            ],
+            infoLink: 'https://en.wikipedia.org/wiki/Common_European_Framework_of_Reference_for_Languages',
+        },
+        seniority: {
```
Comment (kevfarid) [inline src/services/salaries.js]: Use API: https://143.198.62.170/#/Salaries/titles_api_seniority_get

File: src/services/salaries.js [hunk 8]
```diff
@@ -4,7 +4,29 @@ const API_URL = process.env.SALARIES_API_URL;
 
 export const getListByName = (name) => {
     const lists = {
-        english: ['A1', 'A2', 'B1', 'B2', 'C1', 'C2'],
+        english: {
```
Comment (kevfarid) [inline src/services/salaries.js]: Use API: https://143.198.62.170/#/Salaries/titles_api_english_get

**Review by kevfarid — CHANGES_REQUESTED** [review]

**Review by kevfarid — DISMISSED** [review]
Nice, good job

**Review by yadurani — DISMISSED** [review]
LGTM!

**Review by yadurani — APPROVED** [review]
LGTM!

**Pull request merged**